projectq/cengines/_twodmapper.py

```diff
@@ -49,14 +49,18 @@ class GridMapper(BasicMapperEngine):
     |   |
     4 - 5
 
-    The numbers are the mapped qubit ids.
+    The numbers are the mapped qubit ids. The backend might number
+    the qubits on the grid differently (e.g. not row-major), we call these
+    backend qubit ids. If the backend qubit ids are not row-major, one can
+    pass a dictionary translating from our row-major mapped ids to these
+    backend ids.
 
     Note: The algorithm sorts twice inside each column and once inside each
           row.
 
     Attributes:
         current_mapping:  Stores the mapping: key is logical qubit id, value
-                          is mapped qubit id from 0,...,self.num_qubits
+                          are backend qubit ids.
```

Comment:
minor change: "value is backend qubit id"

Reply:
Good catch